scilpy/io/utils.py

```diff
@@ -121,6 +121,16 @@ def add_sh_basis_args(parser, mandatory=False):
                         help=help_msg)
 
 
+def check_sh_basis_choice(sh_basis):
```

Comment:
Can we rename it to ""validate_sh_basis_choice". _Check_ does not feel explicit enough for me